### PR TITLE
fix: single notecard is taking 100% width

### PR DESCRIPTION
**Pull request by phaneendraandukuri**
Reference Issue: https://github.com/twentyhq/twenty/issues/1966

### CODE REVIEW

File: front/src/modules/activities/notes/components/NoteCard.tsx [hunk 1]
```diff
@@ -13,6 +14,12 @@ const StyledCard = styled.div`
   flex-direction: column;
   height: 300px;
   justify-content: space-between;
+
+  @media (min-width: ${MOBILE_VIEWPORT}px) {
+    &:only-child {
+      width: 50%;
+    }
+  }
```
[the remark this answers is not in the document]
Comment (cricket-lover) [reply front/src/modules/activities/notes/components/NoteCard.tsx:22]: Sure, Will make the changes accordingly.

[the remark this answers is not in the document]
Comment (cricket-lover) [reply front/src/modules/activities/notes/components/NoteCard.tsx:22]: @bosiraphael I am thinking to use fixed max-width to each of the child. Do you think that is a good idea?

[the remark this answers is not in the document]
Comment (bosiraphael) [reply front/src/modules/activities/notes/components/NoteCard.tsx]: @cricket-lover I think it would work as well yes! :)

**Comment by charlesBochet** [comment]
@cricket-lover The current PR is empty, do you plan to re-push on it? :)

**Comment by cricket-lover** [comment]
Yes, I will @charlesBochet I will push the changes

**Comment by charlesBochet** [comment]
Ok!

**Comment by cricket-lover** [comment]
@bosiraphael I have made the changes and it is working properly now. Can you please review it?

**Review by bosiraphael — COMMENTED** [review]
Thank you for your contribution @cricket-lover , I left you a comment in my review.

**Comment by charlesBochet** [comment]
@bosiraphael @cricket-lover I will close this PR for now, feel free to re-open it with updated changes :)

**Pull request closed without merging**